cmd/konnectd/serve.go

```diff
@@ -178,3 +190,24 @@ func serve(cmd *cobra.Command, args []string) error {
 	logger.Infoln("serve started")
 	return srv.Serve(ctx)
 }
+
+func envOrDefault(name string, def string) string {
```

Comment:
I think all commandline parameters should use this, right now only two do.

Reply:
The other commandline parameters have now default or no env var defined. So "envOrDefault" wouldn't make sense in this context?